root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -3392,34 +3784,39 @@ const testData = [
     expected_relationship_type: 'otherdatabases',
             expected_clean_url: 'https://ra.co/dj/adamx',
        only_valid_entity_types: ['artist'],
+        expected_favicon_class: 'residentadvisor-favicon',
   },
   {
                      input_url: 'https://ra.co/events/860109#',
              input_entity_type: 'event',
     expected_relationship_type: 'otherdatabases',
             expected_clean_url: 'https://ra.co/events/860109',
        only_valid_entity_types: ['event'],
+        expected_favicon_class: 'residentadvisor-favicon',
   },
   {
                      input_url: 'http://ra.co/labels/2795',
              input_entity_type: 'label',
     expected_relationship_type: 'otherdatabases',
             expected_clean_url: 'https://ra.co/labels/2795',
        only_valid_entity_types: ['label'],
+        expected_favicon_class: 'residentadvisor-favicon',
   },
   {
                      input_url: 'https://ra.co/clubs/5031/events',
              input_entity_type: 'place',
     expected_relationship_type: 'otherdatabases',
             expected_clean_url: 'https://ra.co/clubs/5031',
        only_valid_entity_types: ['place'],
+        expected_favicon_class: 'residentadvisor-favicon',
   },
   {
                      input_url: 'https://ra.co/tracks/544258',
              input_entity_type: 'recording',
     expected_relationship_type: 'otherdatabases',
             expected_clean_url: 'https://ra.co/tracks/544258',
        only_valid_entity_types: ['recording'],
+        expected_favicon_class: 'residentadvisor-favicon',
   },
   {
                      input_url: 'http://www.ra.co/reviews/7636',
```

Comment:
Maybe test RA reviews’ favicon too?

Reply:
I don't think they have a different favicon for reviews? Am I missing something? :)

Reply:
Right. I was initially looking for a favicon test of `kenjimiura.com` when I noticed you added a favicon test to all RA tests but that one; I just wondered why. Now looking at the code again ([commit `86ef099` - `root/static/scripts/common/constants.js` - line 156](https://github.com/metabrainz/musicbrainz-server/pull/2256/commits/86ef0995c19be175295f8b4af667c43ecf087d1f#diff-a0ecd03ab7967706f21d4f2a99034bd55a8daef0272fbace5c6b63d02c1ba01cR156)), setting this favicon doesn’t depend on the path indeed. So maybe it’s not useful to have 7 favicon tests for RA URLs, having 1 might be enough?

Reply:
Oh, now I see :) I added it to all of them but the one with no valid entity types (that's what I generally did elsewhere too) just in case we changed some specific cleanup and it no longer matched the favicon, but admittedly it might be overdoing it :) I can remove it from the rest if desired.